Week-2/article/article.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="styles.css" rel="stylesheet">
+    <title>5 Things to Know About Rattlesnakes and Their Babies</title>
+</head>
+<body>
+    <img class="first_image" src="https://static-resources.zybooks.com/static/UofA_header.svg">
```

Comment:
Always add alt text property for an img tag
https://www.w3schools.com/tags/att_img_alt.asp